clang/test/Modules/pr68702.cpp

```diff
@@ -0,0 +1,65 @@
+// RUN: rm -rf %t
+// RUN: mkdir %t
+// RUN: split-file %s %t
+
+// RUN: %clang_cc1 -std=c++20 -emit-obj -fmodules -fimplicit-module-maps -fmodules-cache-path=%t %t/main.cpp -o %t/main.o
```

Comment:
I thought this is a frontend issue -- do we need `-emit-obj` here?

Reply:
No, we don't need it, you are right - the crash happens during parsing / sema